ngraph/frontend/paddlepaddle/src/node_context.hpp

```diff
@@ -108,6 +112,17 @@ template <>
 inline ngraph::element::Type NodeContext::get_attribute (const std::string& name, const ngraph::element::Type& def) const
 { return node.get_dtype(name, def); }
 
+inline ngraph::element::Type NodeContext::get_out_port_type(const std::string& port_name) const
+{
+    auto types = get_out_port_types(port_name);
+    PDPD_ASSERT(types.size() > 0, "Port has no tensors connected.");
+    ngraph::element::Type res = types[0];
+    for (const auto& type : types) {
```

Comment:
minor: std::equal could be probably used for simplicity

Reply:
std::equal is for comparing 2 vectors, not vector to element. std::find can be used here, not sure if it will be easier to read